modules/installation-uninstall-aws.adoc

```diff
@@ -7,27 +7,87 @@
 
 You can remove a cluster that you installed on Amazon Web Services (AWS).
 
+.Prerequisites
+
+* Deploy an {product-title} cluster.
+* Have a copy of the installation program that you used to deploy the cluster.
+
 .Procedure
 
-. Optionally, from the computer that you used to install the cluster, run the
-following command and record the UUID that it outputs:
+Perform the following steps on the computer that you used to install the cluster.
+
+. Obtain the cluster UUID:
+
+** If you have access to the `oc` command line interface (CLI), log in and run the
+following command:
 +
-[source,bash]
 ----
 $ oc get clusterversion -o jsonpath='{.spec.clusterID}{"\n"}' version
+
+df328c20-0dbd-4d02-a694-30949d7856e7
+----
++
+The command output is the cluster's UUID. You can use this UUID to generate a new
+`metadata.json` file for your cluster, if required. If not all of the cluster
+resources are removed from AWS when you delete the cluster, you can also use
+this UUID to locate them and remove them.
+** If you do not have access to the `oc` CLI but you know the AWS region the
+cluster was deployed to and the cluster ID, use the AWS CLI to obtain it:
++
+[NOTE]
+====
+If you did not install the AWS CLI when you deployed your cluster, see
+link:https://docs.aws.amazon.com/cli/latest/userguide/install-bundle.html[Install the AWS CLI Using the Bundled Installer (Linux, macOS, or Unix)]
+in the AWS documentation.
+====
++
+----
+$ aws ec2 --region <aws_region> describe-vpcs --output json | jq '.Vpcs[] | {"name": ([.Tags[] | select(.Key == "Name") | .Value][0]), "openshiftClusterID": ([.Tags[] | select(.Key == "openshiftClusterID") | .Value][0])} | select(.name |contains("<cluster_name>") )' <1>
+
+{
+  "name": "<cluster_name>.<Route53_zone>",
+  "openshiftClusterID": "df328c20-0dbd-4d02-a694-30949d7856e7"
+}
 ----
+<1> In this command, `<aws_region>` is the AWS region the cluster was deployed
+in, and `<cluster_name>` is the cluster's name.
 +
-If not all of the cluster resources are removed from AWS, you can use this UUID
-to locate them and remove them.
+Locate your cluster's name in the command output and record the
+`openshiftClusterID` value for it. The `openshiftClusterID` value is the same
+as the cluster UUID.
+
+. Locate the `metadata.json` file that describes your cluster. This file is in
+the `installation_directory` directory that you specified when you deployed your
+cluster.
+
+. If you cannot locate the `metadata.json` file, recreate it:
+
+.. Store the cluster name, AWS region, and UUID as environment variables:
++
+----
+$ AWS_REGION=<aws_region> <1>
+$ CLUSTER_NAME=<cluster_name> <2>
+$ CLUSTER_UUID=<cluster_UUID> <3>
+----
+<1> `<aws_region>` is the AWS region the cluster was deployed in.
+<2> `<cluster_name>` is the cluster's name.
+<3> `<cluster_UUID>` is the cluster's UUID that you obtained.
+
+.. Generate the `metadata.json` file for your cluster:
```

Comment:
This command needs to be run in the installation directory, or you need to move the generated `metadata.json` file there.

Reply:
I'm adding this step:
 > If you still have the `installation_directory` directory, change to that directory.